report/Sprint_1/Application.tex

```diff
@@ -0,0 +1,92 @@
+\section{Application}
+During the first sprint, we started development on the application.
+The following features were implemented:
+\begin{itemize}
+    \item Login screen
+    \item Application selection screen
+    \item Create application screen
+    \item Create user screen
+    \item Task dashboard screen
+    \item Create task screen
+\end{itemize}
+
+In this sprint, some of the functionality was mocked, because they were yet to be developed on the backend.
+This was however limited to task functionality, where the rest was actual data transmitted from the API.
+
+\subsection{Design}
+\label{sprint_1_design}
+In the start of this sprint, some views were designed to give an idea of how the application should look.
```

Comment:
Er ikke fan af at du skriver in the start of this sprint, så tit.
Derudover så kunne du godt gå mere i detaljer om hvordan vores process var i at lave mock-ups til prototype og nu til application, hvilket også kan blive set som en advance prototype :D